cmd/sonobuoy/app/results.go

```diff
@@ -235,6 +247,17 @@ func printSinglePlugin(input resultsInput, r *results.Reader) error {
 		}
 		_, err = io.Copy(os.Stdout, fReader)
 		return err
+	} else if input.mode == resultModeReadable {
+		fReader, err := r.PluginResultsReader(input.plugin)
+		if err != nil {
+			return errors.Wrapf(err, "failed to get results reader for plugin %v", input.plugin)
+		}
+		data, err := io.ReadAll(fReader)
```

Comment:
So this is a reasonable first approach but it won't work well when the files are large.

What we need to do is, just like the `dump` mode, is to copy from the reader (which uses a smaller buffer and series of reads rather than one big chunk) but to replace the characters as we copy.

I can give you some tips on this too.